learning_resources/management/commands/backpopulate_oll_files.py

```diff
@@ -0,0 +1,42 @@
+"""Management command for populating OLL course run file data"""
+
+from django.core.management import BaseCommand
+
+from learning_resources.tasks import import_all_oll_files
+from main import settings
+from main.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """Populate OLL course run files"""
+
+    help = "Populate OLL course run files"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-c",
+            "--chunk-size",
+            dest="chunk_size",
+            default=settings.LEARNING_COURSE_ITERATOR_CHUNK_SIZE,
+            type=int,
+            help="Chunk size for batch import task",
+        )
+
+    def handle(self, *args, **options):  # noqa: ARG002
+        """Run Populate OLL course run files"""
+        if not (
+            settings.OLL_LEARNING_COURSE_BUCKET_NAME
+            and settings.OLL_LEARNING_COURSE_BUCKET_PREFIX
```

Comment:
this has open-learning-library/courses/ as a default so you don't have to check for it